Flat/interface/GeneralTree.h

```diff
@@ -309,6 +309,9 @@ class GeneralTree : public genericTree {
       int electronTripleCharge[NLEP];
       float electronCombIso[NLEP];
 
+      int loosePho1SelBit;
```

Comment:
These variables were added manually and not defined in the tree configuration.

Reply:
You are right, I forgot about it. How do I go from the tree configuration to the GeneralTree module? (thanks)

Reply:
You add the lines to the configuration and then call 
```./Flat/bin/generateTreeClass.py --cfg configGeneralTree.cfg```